api/api.py

```diff
@@ -170,14 +169,14 @@ async def get_projects():
     escaped_data = [
         [
             project[0],
-            escape(project[1], quote=False),
-            escape(project[2], quote=False),
-            safe_escape(project[3]),
+            sanitize(project[1]),
+            sanitize(project[2]),
+            sanitize(project[3]),
             project[4],
             project[5],
-            safe_escape(project[6]),
-            safe_escape(project[7]),
-            safe_escape(project[8]),
+            sanitize(project[6]),
+            sanitize(project[7]),
+            sanitize(project[8]),
         ]
```

Comment:
no new fixed length array. Please sanitze the whole project dynamically, so that if it ever becomes one entry longer by changing the SQL query this is automatically handled